SimulatorFM/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "author": {
+    "name": "HuckinB",
+    "id": "228575716214702080"
+  },
+  "service": "SimulatorFM",
+  "description": {
+    "en": "Your Music, For Your Simulation."
+  },
+  "url": "simulator.fm",
+  "version": "1.0.1",
+  "logo": "https://panel.simulator.fm/public/avatars/default.jpg",
+  "thumbnail": "https://i.huckinb.com/t3xXxQc",

```

Comment:
upload to imgur, or if huckinb is your own you can use that.

Reply:
huckinb.com is my domain yes

Reply:
then upload the logo to it aswell

Reply:
For simplicity I uploaded them to both to Imgur

Reply:
Proof of Working: https://i.imgur.com/kBxfJg1.png